zerver/lib/home.py

```diff
@@ -177,6 +178,9 @@ def build_page_params_for_home_page_load(
     )
 
     two_fa_enabled = settings.TWO_FACTOR_AUTHENTICATION_ENABLED and user_profile is not None
+    billing_info = get_billing_info(user_profile) if user_profile else None
```

Comment:
I'm not sure this sort of `else None` logic on `billing_info` makes sense; we should just make `get_billing_info` return an object in the normal format in a prep commit.